xcell/mappers/mapper_DELS.py

```diff
@@ -59,13 +73,25 @@ def _get_catalog(self):
         return cat.as_array()
 
     def get_catalog(self):
+        """
+        Checks if the mapper has already \
+        loaded the chosen bin of the catalog. \
+        If so, it loads it from the save file. \
+        Otherwise, it cuts the original file
+
+        Returns:
+            catalog (Array)
```

Comment:
Are you sure it's an array? (same in the other mappers) It might be a Table from astropy